server/data/assessmentStateService.ts

```diff
@@ -223,7 +224,27 @@ export class AssessmentStateService {
   }
 
   async getWorkingAssessmentVersion(stateKey: StateKey) {
-    return (await this.store.getWorkingAssessment(stateKey)).assessment.version
+    let version: number
+    try {
+      version = (await this.store.getWorkingAssessment(stateKey)).assessment.version
+    } catch (err) {
```

Comment:
I'm not sure how this can throw an error as the assessment object is always defined so maybe we can simplify this down to just `const version = (await this.store.getWorkingAssessment(stateKey)).assessment.version` without the try/catch

Reply:
Done